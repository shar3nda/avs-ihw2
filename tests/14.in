sQA
Gd\CCK0jL
DF|AZ#v[@rw*CIElq~9VmQDc7:?`!LI3UEVG'6A'-tHiobzBHp\7T
HRio
$k