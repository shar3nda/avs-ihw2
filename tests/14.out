Lowercase: 412, uppercase: 418